src/com/hms/flexyosisoftconnector/PayloadBuilder.java

```diff
@@ -312,17 +312,18 @@ public static String getContainerSettingJson(int startTagIndex, int numToProcces
       endTagIndex = TagInfoManager.getTagInfoList().size();
     }
     for (int i = startTagIndex; i < endTagIndex; i++) {
+      if (TagInfoManager.getTagInfoList().get(i) != null) {
+        String tagName = ((TagInfo) TagInfoManager.getTagInfoList().get(i)).getName();
 
-      String tagName = ((TagInfo) TagInfoManager.getTagInfoList().get(i)).getName();
+        // after the first tag, separate by comma
+        if (i > startTagIndex) {
+          payload.append(",");
+        }
 
-      // after the first tag, separate by comma
-      if (i > startTagIndex) {
-        payload.append(",");
+        payload.append(
+            "{" + "\"id\": \"" + tagName + "\"," + "\"typeid\": \"" + typeID + "\"" + "}");
       }
-
-      payload.append("{" + "\"id\": \"" + tagName + "\"," + "\"typeid\": \"" + typeID + "\"" + "}");
     }
-
```

Comment:
Whitespace change

Reply:
that's now a } and it looks weird with more white space changes around it. It's a bit hidden in that diff

Reply:
oh my mistake, i see it